project/Dependencies.scala

```diff
@@ -23,7 +23,7 @@ object Dependencies {
   val nettyVersion = "3.10.6.Final"
   val jacksonVersion = "2.10.0"
   val jacksonDatabindVersion = "2.10.0"
-  val protobufJavaVersion = "3.9.2"
+  val protobufJavaVersion = "3.10.0"
```

Comment:
will must also update `protocVersion := "3.9.0"`, and maybe even recompile all protos to see if there are any changes